mono/metadata/locales.c

```diff
@@ -44,6 +44,12 @@
 
 #undef DEBUG
 
+#define mono_handle_setval(handle,field,value) MONO_HANDLE_RAW (handle)->field = (value)
```

Comment:
So the reason why `MONO_HANDLE_SETVAL` is more complicated than this and has a `TYPE` argument is because we want to be safe even if I write `mono_handle_setval (handle, fieldName, some_function())` and `some_function` triggers a GC.  We don't want the raw handle exposed until after `(value)` is evaluated.

If you want to do this locally in locales.c, please add a comment to discourage anyone from moving it to `handle.h`